rented-properties/src/main/java/org/egov/cpt/util/FileStoreUtils.java

```diff
@@ -41,4 +51,31 @@ public String fetchFileStoreUrl(ExcelSearchCriteria searchCriteria) {
 		return responseMap;
 	}
 
+	@SuppressWarnings("unchecked")
+	public String fetchFileStoreId(File file, Property property) {
```

Comment:
Let's return a `List<HashMap<String, String>>` as a return type and ultimate send that to the client.

Reply:
returning list as response